Scripts/RemediationScripts/Remediate-CdnFrontDoorMinTLSVersion.ps1

```diff
@@ -550,10 +493,10 @@ function Set-FrontDoorRequiredTLSVersion
     {
         if(-not $SkipBackup)
         {
-            Write-Host "Backing up Front Doors details to [$($backupFolderPath)]..." -ForegroundColor $([Constants]::MessageType.Info)
+            Write-Host "Backing up Front Door CDN Profiles details to [$($backupFolderPath)]..." -ForegroundColor $([Constants]::MessageType.Info)
             $BackupFile = "$($backupFolderPath)\CdnFrontDoorsWithoutReqMinTLSVersion.csv"
-            $NonCompliantFrontDoorEndpoints | Export-CSV -Path $BackupFile -NoTypeInformation
-            Write-Host "Front Doors details have been backed up to [$($BackupFile)]." -ForegroundColor $([Constants]::MessageType.Update)
+            $NonCompliantFrontDoorDomains | Export-CSV -Path $BackupFile -NoTypeInformation
+            Write-Host "Front Door CDN Profiles details have been backed up to [$($BackupFile)]." -ForegroundColor $([Constants]::MessageType.Update)
             Write-Host $([Constants]::SingleDashLine)
         }
```

Comment:
$SkipBackup parameter is not used now a days. See the recent BRSes. And modify the section accordingly.

Reply:
Also line 517: It should be double dash lines